src/actinia_tiling_plugin/api/merge/patch_merge.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Copyright (c) 2022 mundialis GmbH & Co. KG
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Grid Tiling Class
+"""
+
+__license__ = "GPLv3"
+__author__ = "Anika Weinmann"
+__copyright__ = "Copyright 2022 mundialis GmbH & Co. KG"
+__maintainer__ = "mundialis GmbH % Co. KG"
+
+from flask import make_response, jsonify
+from flask_restful_swagger_2 import swagger
+import pickle
+from copy import deepcopy
+
+from actinia_core.processing.actinia_processing.ephemeral.\
+    persistent_processing import PersistentProcessing
+from actinia_core.rest.base.resource_base import ResourceBase
+from actinia_core.core.common.redis_interface import enqueue_job
+# from actinia_core.core.common.process_chain import ProcessChainConverter
+from actinia_core.processing.actinia_processing.persistent.mapset_management \
+    import (
+        PersistentMapsetDeleter,
+        PersistentMapsetUnlocker,
```

Comment:
Please add this information (and maybe other relevant from this repository?) To the README in actinia_core: https://github.com/mundialis/actinia_core/blob/main/src/actinia_core/README.md

Reply:
Done: https://github.com/mundialis/actinia_core/pull/342